third_party/move/move-vm/types/src/delayed_values/derived_string_snapshot.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) The Move Contributors
```

Comment:
I'm a bit unsure, but is "string snapshot" a concept that belongs to the Move VM layer?

Reply:
This is a helper file for processing strings for snapshots, the problem I had was that serialization is defined inside `values` crate, so I need access to these functions. Ideally, custom serializers/deserializers should live outside, and so we can shift all this code to aptos layer